endive-playground/src/index.html

```diff
@@ -21,11 +23,19 @@
       <textarea id="endivecodearea" style="height: 45vh; width: 49.3%">
 Endive generated code goes here</textarea
       >
-      <textarea id="latexcodearea" style="height: 45vh; width: 49.3%">
-LaTeX generated code goes here</textarea
+      <div id="latexcodearea" style="height: 45vh; width: 49.3%">
+LaTeX generated code goes here</div
       >
     </main>
 
+    <script>
+      MathJax = {
+        tex: {
+          inlineMath: [['$', '$']]
+        }
+      };
+    </script>
+    <script type="text/javascript" id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
```

Comment:
As we have a build system already, we should probably add `mathjax` as a dependency to our NPM package and import it as an NPM package so that we don't depend on jsdelivr and we can bundle MathJax and our code in a single script.